plugins/PrivacyManager/API.php

```diff
@@ -400,6 +409,36 @@ public function executeDataPurge(
         }
     }
 
+    /**
+     * @internal
+     */
+    public function getComplianceStatus(string $idSite, string $complianceType): array
+    {
+        if ($this->featureFlagManager->isFeatureActive(PrivacyCompliance::class) === false) {
+            return [];
+        }
+
+        if ($complianceType !== 'cnil') {
+            return [];
+        }
+
+        Piwik::checkUserHasSuperUserAccess();
```

Comment:
We could run `Db::query(sprintf('UPDATE %s SET `superuser_access` = ? WHERE `login` = ?', Common::prefixTable('user')), [0, 'admin']);` to remove the admin access and then restore it after the test.